src/Ens/JobeetBundle/Entity/Job.php

```diff
@@ -0,0 +1,493 @@
+<?php
+
+namespace Ens\JobeetBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Job
+ */
+class Job
+{
+    /**
+     * @var integer
+     */
+    private $id;
+
+    /**
+     * @var string
+     */
+    private $type;
+
+    /**
+     * @var string
+     */
+    private $company;
+
+    /**
+     * @var string
+     */
+    private $logo;
+
+    /**
+     * @var string
+     */
+    private $url;
+
+    /**
+     * @var string
+     */
+    private $position;
+
+    /**
+     * @var string
+     */
+    private $location;
+
+    /**
+     * @var string
+     */
+    private $description;
+
+    /**
+     * @var string
+     */
+    private $how_to_apply;
+
+    /**
+     * @var string
+     */
+    private $token;
+
+    /**
+     * @var boolean
+     */
+    private $is_public;
+
+    /**
+     * @var boolean
+     */
+    private $is_activated;
+
+    /**
+     * @var string
+     */
+    private $email;
+
+    /**
+     * @var \DateTime
+     */
+    private $expires_at;
+
+    /**
+     * @var \DateTime
+     */
+    private $created_at;
+
+    /**
+     * @var \DateTime
+     */
+    private $updated_at;
+
+    /**
+     * @var \Ens\JobeetBundle\Entity\Category
+     */
+    private $category;
+
+
```

Comment:
@PhotonSapiens 
this is not something important, but to keep code beautiful - always use ` line between different properties / functions
You sometimes use 0 and sometimes 2